apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -701,14 +702,11 @@ public void downloadBlob(String container, String name, File destination, Execut
          Futures.getUnchecked(Futures.allAsList(results));
 
       } catch (IOException e) {
-         // cleanup, attempt to delete large file
-         if (raf != null) {
-            try {
-               raf.close();
-            } catch (IOException e1) {}
-         }
+         Closeables2.closeQuietly(raf);
```

Comment:
That won't work, it's trying to delete the file for cleanup; it can't do that if it' open. Which kinda sucks.